tests/installation/livecdreboot.pm

```diff
@@ -15,8 +15,8 @@ use bmwqemu ();
 
 sub run() {
     my $self = shift;
-    # NET isos are slow to install
-    my $timeout = 2000;
+    # NET isos are slow to install, slow machines will take long to install
+    my $timeout = 4000;
```

Comment:
If the machine is slow, it should really define TIMEOUT_SCALE. I'm against slowing down every failure for another half an hour